lib/app-express.js

```diff
@@ -149,27 +234,59 @@ module.exports = function (app) {
       }
 
       // check to see if org is a dev hub
-      const conn = new jsforce.Connection({
+      let connLocal = new jsforce.Connection({
         instanceUrl: payload.instance_url,
         accessToken: payload.access_token
       });
 
-      conn.identity((err, identity) => {
+      connLocal.identity((err, identity) => {
         if (err) {
           return console.error(err);
         }
+        console.log(identity);
```

Comment:
This will put info into logs that shouldn't be persisted there

Reply:
agreed, done.